docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+    web:
+        image: blurts-server
+        ports:
+            - "6060:6060"
+        environment:
+            - DATABASE_URL="postgres://postgres@db:5432/postgres"
```

Comment:
I'm getting a weird (mismatch?) error with this ...
```
 knexfile.js, AppConstants.DATABASE_URL:  "postgres://postgres@db:5432/postgres"
 module.exports:  { client: 'pg',
   connection: '"postgres://postgres@db:5432/postgres"',
   pool: { min: 2, max: 10 } }
 Attempting to get SMTP credentials from environment...
 Listening on 6060
...
 (node:7) UnhandledPromiseRejectionWarning: Error: connect ECONNREFUSED 127.0.0.1:5432
     at Object._errnoException (util.js:992:11)
     at _exceptionWithHostPort (util.js:1014:20)
     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1186:14)
 (node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an
nction without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
 (node:7) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are
```

The [docker-compose links docs](https://docs.docker.com/compose/networking/#links) say I should be able to do this?